bootstrap.yaml

```diff
@@ -0,0 +1,23 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 127.0.0.1, port_value: 9901 }
```

Comment:
If this runs in a container, it's probably best to bind to `0.0.0.0` instead of localhost.